SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -140,6 +148,14 @@ void USpatialGameInstance::TryConnectToSpatial()
 			}
 		}
 	}
+#if TRACE_LIB_ACTIVE
+	else
+	{
+		// Setup native worker name
+		FString WorkerName = FString::Printf(TEXT("N:%s:%s"), *SpatialWorkerType.ToString(), *FGuid::NewGuid().ToString(EGuidFormats::Short));
```

Comment:
Note that `workerType` command-line option needs to be set even when running native which might not be obvious to everyone.